src/backend/cgcod.c

```diff
@@ -726,7 +726,7 @@ code *prolog()
     assert(usedalloca != 1);
     AllocaOff = alignsection(usedalloca ? (Foff - REGSIZE) : Foff, REGSIZE, bias);
 
-    CSoff = alignsection(AllocaOff - cstop * REGSIZE, REGSIZE, bias);
+    CSoff = alignsection(AllocaOff - (int)cstop * REGSIZE, REGSIZE, bias);
```

Comment:
??? Please reread my comment above, there is no issue with cstop.

The problem is that `cstop` being unsigned causes the (possibly negative, though typed `target_size_t`) argument to `alignsection` not to be sign-extended, so variables `CSoff`, `NDPoff` and `localsize` can get bad values.